css/cardlist.css

```
@@ -27,7 +27,7 @@
   border-top-right-radius: 2px;
 }
 
-.card-heading h4 {
+.card-heading h2 {
```

The change from h4 to h2 makes the heading take up more space (because `line-height` is larger for h2), with the same font size (because of the rule below). I would suggest to preserve the old line spacing (or even set it to slightly smaller like 1.2).

```suggestion
.card-heading h2 {
     line-height: 1.22;
```